src/sfctl/custom_chaos_schedule.py

```diff
@@ -0,0 +1,181 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for the Service Fabric chaos schedule test service"""
+
+def parse_time_of_day(time_of_day):
+    """Parse a TimeOfDay from string"""
+    from azure.servicefabric.models.time_of_day import (
+        TimeOfDay
+    )
+
+    if time_of_day is None:
```

Comment:
Maybe consider just `if time_of_day`, I think you want to return `None` on empty string.